### ISSUE TITLE
[sysrst_ctrl] Adding dedicated WP_L function

### ISSUE THREAD

**Issue by tjaychen** [lowRISC/opentitan#7797]
This is something that is missing from the current design. 

Two functions from sysrst_ctrl need to be dedicated and active immediately when the device is powered up.
These are `EC_RST_L` and `FLASH_WP_L`.  `EC_RST_L` is already implemented, however `FLASH_WP_L` is missing. 

`FLASH_WP_L` behavior is as follows:

- at power up, drive 0 onto the dedicated pin
- afterwards, keep driving 0 until configured by software to do otherwise. 

Basically, `FLASH_WP_L` behaves "almost" like a GPIO but must be actively driven immediately after power-up.

**Comment by toshaq** [comment]
LGTM. This should be the correct behavior.

**Comment by tjaychen** [comment]
@cindychip @sriyerg @weicaiyang @matutem FYI

**Comment by sriyerg** [comment]
Ack. Will update the chip testplan accordingly.

**Comment by sriyerg** [comment]
See PR #7800 - let me know if anything needs to change.

**Comment by msfschaffner** [comment]
This has been implemented as part of #7829.
